main.py

```diff
@@ -49,21 +50,22 @@ def make_graph(percent: float, ip_blocks: str, length: int = GRAPH_LENGTH) -> st
     return graph
 
 
-def get_stats() -> str:
+def get_stats(range: str = 'last_7_days') -> str:
     '''Gets API data and returns markdown progress'''
     encoded_key: str = str(base64.b64encode(waka_key.encode('utf-8')), 'utf-8')
     data = requests.get(
-        f"{api_base_url.rstrip('/')}/v1/users/current/stats/last_7_days",
+        f"{api_base_url.rstrip('/')}/v1/users/current/stats/{range}",
```

Comment:
![image](https://user-images.githubusercontent.com/38415384/154844401-461f9063-30c4-4e60-9732-ae1b5fa7e259.png)

There's no check made if an invalid range is passed..

Reply:
I think it's the user's responsibility to provide a valid value, since the Wakatime API may change over time.
However, I agree to provide a more specific error message, if a KeyValue exception has been raised. Does that sound okay?

Reply:
Another reason to not check the value is that since the wakatime API docs are not completed (for example, `all time` is a valid value but not found in the docs), we may limit what the users can do with this action.

Reply:
I see..